src/MoBi.UI/Views/AddBuildingBlocksToModuleView.cs

```diff
@@ -32,13 +44,46 @@ protected override void StartValueCheckChanged(bool enabled, LayoutControlItem n
       public void AttachPresenter(IAddBuildingBlocksToModulePresenter presenter)
       {
       }
+
+      public void adjustViewForOnlyParameterValues()
+      {
+         ShowStartValueNameControls();
+         cbParameterValues.Checked = true;
+         cbParameterValues.Enabled = false;
+
+         hideAllItemsExcept(new List<BaseLayoutItem> { parameterValuesNameItem, parameterValuesItem });
+      }
+
+      public void adjustViewForOnlyInitialConditions()
+      {
+         ShowStartValueNameControls();
+         cbInitialConditions.Checked = true;
+         cbInitialConditions.Enabled = false;
+
+         hideAllItemsExcept(new List<BaseLayoutItem> { initialConditionsNameItem, initialConditionsItem });
+      }
+
+      private void adjustHeight(int heightAdjustment)
+      {
+         Size = new Size(Width, Height + heightAdjustment);
+      }
+
+      private void hideAllItemsExcept(List<BaseLayoutItem> itemsToShow)
+      {
+         var initialGroupHeight = createBuildingBlocksGroup.Size.Height;
```

Comment:
what's with this height here? I don't quite get it

Reply:
That's the height of the layout group before hiding many items.

Later we take the difference and then apply that difference to the dialog height to adjust.